### PR TITLE
feat(localization): add Traditional Chinese language support

### PR DESCRIPTION
**Pull request by goodair220917**
## Summary

Adding Traditional Chinese translation for most of the texts. 3 original translation values of "Show Agent Setting", "Show Completion Setting" and "Plugins" can't be find in Eng.tsx for translating so these are left as English translation, this issue is also found in other translations available in LibreChat now. 

## Change Type

- [x]  New feature (non-breaking change which adds functionality)

## Testing

Please describe your test process and include instructions so that we can reproduce your test. If there are any important variables for your testing configuration, list them here.

1. Open Setting menu
2. Select 繁體中文 as the language
3. Done

### **Test Configuration**:

* Language: Traditional Chinese

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

File: client/src/localization/languages/ZhTraditional.tsx [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+  com_nav_user: '用戶',
+  com_nav_clear_conversation: '清除對話',
+  com_nav_clear_conversation_confirm_message:
+    '您確定要清除所有對話嗎？此操作不可逆轉。',
+  com_nav_help_faq: '幫助與常見問題解答',
+  com_nav_settings: '設置',
+  com_nav_search_placeholder: '搜索訊息',
+  com_nav_setting_general: '一般',
+  com_nav_setting_data: '數據控制',
+  com_nav_language: '語言',
```
Comment (berry-13) [inline client/src/localization/languages/ZhTraditional.tsx]: can you remove this com_nav_language translation?

**Review by berry-13 — COMMENTED** [review]
You also need to add "traditional Chinese" (in your language) to the English translation. It should look something like this:

com_nav_lang_traditionalchinese: '繁體中文'

File: client/src/localization/languages/ZhTraditional.tsx (hunk 1 above)
Comment (goodair220917) [reply client/src/localization/languages/ZhTraditional.tsx]: done

**Review by berry-13 — APPROVED** [review]
Perfect!

**Pull request merged**